### ISSUE TITLE
Add Priority Label to Issues

### ISSUE THREAD

**Issue by danielwiktorczyk** [danielwiktorczyk/Concordia_Course_Builder#36]
**Context**

As a team member, I want to know what priority a given issue has so that I know which issues need to be completed first.

**Requirements**

All completed and future issues are labeled with the following schema:
- P0 tag for highest priority (be done ASAP)
- P1 tag for medium priority (be done before sprint ends)
- P2 tag for low priority  (be done in a future sprint)
- P3 tag for lowest or no priority (may be done in future sprint)

**Comment by danielwiktorczyk** [comment]
Priority labels have been added to existing and completed issues accordingly
